test/health_check_test_case.py

```diff
@@ -84,6 +89,41 @@ def test_health_endpoint_keys(self):
                         self.assertEqual(200, response.status_code)
                         self.assertEqual(expected_response, response.json())
 
+    @mock_s3
+    @mock_sts
+    @mock_sqs
+    def test_cached_health(self):
+        storage_service = StorageService()
+        storage_service.create_bucket()
+        # No health object is available in S3 bucket, yielding an error
+        with ResponsesHelper() as helper:
+            helper.add_passthru(self.base_url)
+            response = requests.get(self.base_url + '/health/cached')
+            self.assertEqual(500, response.status_code)
+            self.assertEqual({'Code': 'InternalServerError',
+                              'Message': 'An internal server error occurred.'}, response.json())
+
+        # A successful response is obtained when all the systems are functional
+        self._create_mock_queues()
+        endpoint_states = self._make_endpoint_states(self.endpoints)
+        app = load_app_module(self.lambda_name())
+        with ResponsesHelper() as helper:
+            helper.add_passthru(self.base_url)
+            self._mock_service_endpoints(helper, endpoint_states)
```

Comment:
… so this line would read 

```suggestion
            with self._mock_service_endpoints(helper, endpoint_states):
```